resources/metadata/protecodeExecuteScan.yaml

```diff
@@ -199,6 +199,21 @@ spec:
           - type: vaultSecret
             name: protecodeVaultSecretName
             default: protecode
+      - name: userApiKey
```

Comment:
Will the `userApiKey` come from a secure store in Jenkins or from Vault? Then you would need to add a secret into the respective section of this yaml.